images/README.md

```diff
@@ -29,8 +29,9 @@ bit more on that [in the cloud-build docs][gcb_images].
 
 ## Currently used images
 
-Image                                     | used in/by
-:---:                                     | --
-[k8s-cloud-builder](./k8s-cloud-builder/) | The "main" image, [`anago`](../anago) runs with on cloud-build (submitted via [`gcbmgr`](../gcbmgr))
+| Image                                     | used in/by                                                                                           |
+| ----------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [k8s-cloud-builder](./k8s-cloud-builder/) | The "main" image, [`anago`](../anago) runs with on cloud-build (submitted via [`gcbmgr`](../gcbmgr)) |
+| [prow](./prow)                            | The prow image used for CI testing                                                                   |
```

Comment:
Can we name the image `bazel`? I can easily imagine a world, where we (releng / k/release) maintain other images which we run on prow.

Reply:
Sure, I rename it to `bazel`.